### PR TITLE
manifests: Upgrade cert-manager API from v1alpha2 to v1

### PR DESCRIPTION
**Pull request by haoxins**

### CODE REVIEW

File: manifests/v1beta1/installs/katib-cert-manager/certificate.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: cert-manager.io/v1alpha2
+apiVersion: cert-manager.io/v1
```
Comment (andreyvelich) [inline manifests/v1beta1/installs/katib-cert-manager/certificate.yaml:1]: Does it require any additional API changes after switching from `v1alpha2` to `v1` ?

cc @yanniszark @kimwnasptd

**Review by andreyvelich — COMMENTED** [review]
Thank you for this update!
I left comment.

**Comment by haoxins** [comment]
I also updated the kserve manifests https://github.com/kserve/kserve/pull/1929

File: manifests/v1beta1/installs/katib-cert-manager/certificate.yaml (hunk 1 above)
Comment (haoxins) [reply manifests/v1beta1/installs/katib-cert-manager/certificate.yaml:1]: We are using `cert-manager` `v1.3` in the manifests repo.

The `cert-manager` deprecated the following cert-manager APIs from `v1.4`

```
cert-manager.io/v1alpha2
cert-manager.io/v1alpha3
cert-manager.io/v1beta1
acme.cert-manager.io/v1alpha2
acme.cert-manager.io/v1alpha3
acme.cert-manager.io/v1beta1
```

https://cert-manager.io/docs/release-notes/release-notes-1.4/

And the latest `cert-manager` only supports `v1` now.

**Comment by haoxins** [comment]
is this ready to merge?

**Comment by andreyvelich** [comment]
Sorry for the late review @haoxins.
/lgtm
/approve
/cc @kimwnasptd

**Pull request merged**